deploy_day2_workers.yml

```diff
@@ -26,4 +26,5 @@
 - hosts: bastion
```

Comment:
Is there a reason not to add the check to the previous play as well?

Reply:
Because hosts effectively does that when anyway if there are no hosts defined for it the play will be skipped.